index.js

```diff
@@ -155,15 +156,15 @@ function generateProperties(options) {
   }, []);
 }
 
-function getJenkinsClassname (test) {
+function getJenkinsClassname (test, options) {
   debug('Building jenkins classname for', test);
   var parent = test.parent;
   var titles = [];
   while (parent) {
     parent.title && titles.unshift(parent.title);
     parent = parent.parent;
   }
-  return titles.join('.');
+  return options.jenkinsClassnamePrefix + titles.join('.');
```

Comment:
It would be good to use `options.suiteTitleSeparatedBy` rather than having to explicitly add it in options.jenkinsClassnamePrefix